zizmor.yml

```diff
@@ -1,6 +1,7 @@
 rules:
   dangerous-triggers:
     ignore:
+      - coverage_comment.yml
```

Comment:
Now that we have two workflows, can we use the `workflow_call` [trigger](https://docs.github.com/en/actions/how-tos/reuse-automations/reuse-workflows) to avoid this ignore?